electron/main.ts

```diff
@@ -336,7 +333,12 @@ ipcMain.on('getLog', (event, appName) =>
   openUrlOrFile(`${heroicGamesConfigPath}${appName}-lastPlay.log`)
 )
 
+/*
+  Used to remove the game folder if someone stops game installation and chooses not to keep the files
```

Comment:
This was fixed here: https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/404
Unless the issue was introduced again.

Reply:
Weird, I just cloned main from before the merge of the logging pr so it should have been there 😬